examples/macros/test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const test = require('ava');
+const { sum } = require('.');
+
+function macro(t, a, b, expected) {
+	t.is(sum(a, b), expected);
+}
+
+macro.title = (providedTitle = '', a, b, expected) => `${providedTitle} ${a}+${b} = ${expected}`.trim();
```

Comment:
@novemberborn linting fails on the `providedTitle = ''`, you want me to remove it and use `${providedTitle || ''}` instead?

Reply:
I removed the default value and used `|| ''`.